leetCode.py

```diff
@@ -0,0 +1,25 @@
+"""
+hash each element in J and put this in a set
+go through S and see if they're in our set
+if it's in the set, then count it 
+return the count 
+"""
+
+"""
+Store elements of J in a set
+Iterate through S
+for each elem in S
+  if elem is in set J
+    then increment result
+return result
+
+O(n): n == length of the S
+O(length of j)
+"""
+class Solution:
```

Comment:
Love that you're adding external resources into your project and learning! Yes!

Type hints are a very powerful tool in Python, that allows it to extend a little into true functional programming (typically there's a type system that will allow your compiler (though Python is _interpreted_), to predict what your code should and shouldn't be doing _without you explicitly stating it_), and it's just great to have as a reference.